wavefront/server/modules/voice_agents_module/voice_agents_module/services/tool_service.py

```
@@ -0,0 +1,538 @@
+            Created VoiceAgentTool instance
+
+        Raises:
+            ValueError: If tool with same name exists
+        """
+        try:
+            # Check if tool with same name exists
+            existing = await self.tool_repository.find_one(
+                name=payload.name, is_deleted=False
+            )
+            if existing:
+                raise ValueError(f'Tool with name "{payload.name}" already exists')
```

Ruff (0.14.13)

Abstract `raise` to an inner function

(TRY301)